src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -75,16 +75,37 @@ public class HomeCard extends UiPart<Region> {
     private TranslateTransition moveTransition;
     private TranslateTransition moveBackTransition;
     private FadeTransition fadeInTransition;
+    private DateTimeFormatter timeFormatter;
+    private DateTimeFormatter secondFormatter;
+    private DateTimeFormatter dateFormatter;
 
     /**
      * Creates a {@code HomeCard} with the given Person List.
      */
     public HomeCard(ObservableList<Person> personList) {
         super(FXML);
         this.personList = personList;
-        DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
-        DateTimeFormatter secondFormatter = DateTimeFormatter.ofPattern(":ss");
-        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("EEEE, MMMM d");
+        setUpTimeline();
+        contactTop.setText(CONTACT_TOP);
+        contactBottom.setText(CONTACT_BOTTOM);
+        contactAmount.setText(getContactCount().toString());
+        XYChart.Series<String, Number> series = new XYChart.Series<>();
+        series.getData().add(new XYChart.Data<>(MONEY_OWED_LABEL, getTotalDebt()));
+        series.getData().add(new XYChart.Data<>(OWED_MONEY_LABEL, getTotalCredit()));
+        chart.getData().add(series);
+        chart.setCategoryGap(CATEGORY_GAP);
+        chart.setBarGap(BAR_GAP);
+        chart.setLegendVisible(false);
+        chartLabel.setText(CHART_LABEL);
+        availableToday.setText(AVAILABLE_LABEL);
```

Comment:
Do you think this part can be extracted out so that your constructor follows SLAP?